### DISCUSSION TITLE
How to disconnect client from host side?

### DISCUSSION THREAD

**Discussion by bikeking8** [LizardByte/community#116] [Sunshine] [answered]
If I want to ensure my friend's moonlight client is disconnected at his place, can I just use the "Unpair All" button in troubleshooting on the localhost Sunshine config, or would that require them to add my PC again in Moonlight and send the pin etc each and every time? Unless I'm missing it, I don't really see an intuitive way to see if any client is connected.

**Comment by ReenigneArcher** [comment, marked as answer]
Unpair all will require you to redo the pairing process.

The terminal/logs should tell you if anyone is connected. You can just kill the sunshine process if you don't want anyone connected. How you do that will depend on how you're running sunshine and what OS you're on.

**Reply by lastmetroid** [reply]
Please implement a "disconect client(s)" in the Sunshine host systray icon, which just ends the stream, not unpairs the client. That would be really handy.

**Reply by ReenigneArcher** [reply]
"Restart" will do that...

**Reply by lastmetroid** [reply]
Not a solution because of this bug:

https://github.com/LizardByte/Sunshine/issues/2515

It happens often randomly you cant click on the Sunshine systray anymore. It happened right now for me again, left right click on Sunshine systray does nothing and you cant access Restart menu. The issue also happens randomly when connecting not even after restarting Systray.

Also Restarting the entire Sunshine process is not a good way to do this, please also add a disconect nect to Restart in the menu and please fix the Systray issue.

Issue just seems to happen if youre connected through Moonlight. Going on the host itself locally it works fine to click on systray, or using VNC.

**Comment by bikeking8** [comment]
Apologies, didn't give enough detail. I'm on Windows 10, and stopping both processes seems to have worked for disconnect. The terms "unpair" or "disconnect" didn't come up in a search of the Sunshine documentation, so I had to resort to this method. Thanks again!
